src/functionalTest/resources/features/F-004/S-040.td.json

```diff
@@ -0,0 +1,69 @@
+{
+  "_guid_": "S-040",
+  "_extends_": "F-004_Test_Base_data",
+  "title": "must successfully upload document with binary content",
+  "specs": [
+    "contains the metadata for the document uploaded above"
+  ],
+  "request": {
+    "headers": {
+      "caseTypeId": "BEFTA_CASETYPE_2_2",
+      "jurisdictionId": "BEFTA_JURISDICTION_2"
+    },
+    "body": {
+      "arrayInMap": [
+        {
+          "key": "classification",
+          "value": "PUBLIC"
+        },
+        {
+          "key": "roles",
+          "value": "caseworker"
+        },
+        {
+          "key": "files",
+          "filePath": "CCD_BEFTA_JURISDICTION2.xlsx"
+        }
+      ]
+    }
+  },
+  "expectedResponse": {
+    "_extends_": "Common_200_Response",
+    "body": {
+      "_embedded": {
```

Comment:
why would we put the documents collection inside an "_embedded" block?

Reply:
The existing document upload endpoint from DM store has this behaviour, so we kept it same just to avoid any issues on the services side where existing DM endpoint is going to be replaced with case-document specific endpoint.

Reply:
response structure will be improved as discussed today.